src/makeRobot.js

```diff
@@ -37,8 +37,74 @@
  *
  * @return {Robot}
  */
+
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name: name,
+    wheels: wheels,
+    version: version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+    goForward(value) {
+      if (value) {
+        if (value > 0) {
+          this.coords.y = this.coords.y + value;
+        }
+      } else {
+        this.coords.y++;
+      }
```

Comment:
lets avoid nested if constructions
just set default value and if it's less then zero - ignore it
```suggestion
    goForward(value = 1) {
        if (value > 0) {
          this.coords.y += value;
        }
  }
```